<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>db_window</class>
 <widget class="QMainWindow" name="db_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="delete_subject">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/icons/delete.xpm</normaloff>:/icons/icons/delete.xpm</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="move_down">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/icons/down.xpm</normaloff>:/icons/icons/down.xpm</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="move_up">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/icons/up.xpm</normaloff>:/icons/icons/up.xpm</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="index_name"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QTableWidget" name="subject_list">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>4</verstretch>
               </sizepolicy>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>true</bool>
              </attribute>
             </widget>
             <widget class="QTextBrowser" name="report">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGraphicsView" name="vbc_view"/>
           </item>
           <item>
            <widget class="QScrollBar" name="slice_pos">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="view_x">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/icons/sag.xpm</normaloff>:/icons/icons/sag.xpm</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="x_pos"/>
             </item>
             <item>
              <widget class="QRadioButton" name="view_y">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/icons/coronal.xpm</normaloff>:/icons/icons/coronal.xpm</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="y_pos"/>
             </item>
             <item>
              <widget class="QRadioButton" name="view_z">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/icons/axial.xpm</normaloff>:/icons/icons/axial.xpm</iconset>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="z_pos"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Zoom</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="zoom">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="minimum">
                <double>0.100000000000000</double>
               </property>
               <property name="maximum">
                <double>20.000000000000000</double>
               </property>
               <property name="value">
                <double>4.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File&amp;</string>
    </property>
    <addaction name="actionAdd_DB"/>
    <addaction name="actionAdd_Database"/>
    <addaction name="separator"/>
    <addaction name="actionSave_DB_as"/>
    <addaction name="actionCurrent_Subject"/>
    <addaction name="actionAll_Subjects"/>
    <addaction name="actionSave_Subject_Name_as"/>
    <addaction name="action_Save_R2_values_as"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Demographics"/>
    <addaction name="actionSave_Demographics"/>
    <addaction name="actionSave_DemoMatched_Image_as"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCalculate_change"/>
    <addaction name="actionSelect_Subjects"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_DB_as">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save DB as...</string>
   </property>
  </action>
  <action name="actionCalculate_change">
   <property name="text">
    <string>Logitudinal scans...</string>
   </property>
  </action>
  <action name="actionSave_Subject_Name_as">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save Subject Name as...</string>
   </property>
  </action>
  <action name="action_Save_R2_values_as">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save R2 values as...</string>
   </property>
  </action>
  <action name="actionAdd_DB">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/add.xpm</normaloff>:/icons/icons/add.xpm</iconset>
   </property>
   <property name="text">
    <string>Add Subjects...</string>
   </property>
  </action>
  <action name="actionSelect_Subjects">
   <property name="text">
    <string>Select Subjects...</string>
   </property>
  </action>
  <action name="actionCurrent_Subject">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save Subject Image as...</string>
   </property>
  </action>
  <action name="actionAll_Subjects">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save All Subjects Image as...</string>
   </property>
  </action>
  <action name="actionSave_Demographics">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save Demographics...</string>
   </property>
  </action>
  <action name="actionOpen_Demographics">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/open.xpm</normaloff>:/icons/icons/open.xpm</iconset>
   </property>
   <property name="text">
    <string>Open Demographics...</string>
   </property>
  </action>
  <action name="actionSave_DemoMatched_Image_as">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
   </property>
   <property name="text">
    <string>Save Regressed Image as...</string>
   </property>
  </action>
  <action name="actionAdd_Database">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/add.xpm</normaloff>:/icons/icons/add.xpm</iconset>
   </property>
   <property name="text">
    <string>Add Database...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
